sdk/eventhub/event-hubs/src/eventHubConsumerClientModels.ts

```diff
@@ -1,7 +1,53 @@
 
 import { CloseReason, PartitionContext, EventProcessorOptions, EventProcessorBatchOptions } from './eventProcessor';
-import { PartitionCheckpointer } from './eventHubConsumerClient';
 import { ReceivedEventData } from './eventData';
+import { EventPosition } from './eventPosition';
+
+/**
+ * Allow for checkpointing
+ */
+export interface PartitionCheckpointer {
+  /**
+   * Updates the checkpoint using the event data.
+   *
+   * A checkpoint is meant to represent the last successfully processed event by the user from a particular
+   * partition of a consumer group in an Event Hub instance.
+   *
+   * @param eventData The event that you want to update the checkpoint with.
+   * @return Promise<void>
+   */
+  updateCheckpoint(eventData: ReceivedEventData): Promise<void>;
+  /**
+   * Updates the checkpoint using the given offset and sequence number.
+   *
+   * A checkpoint is meant to represent the last successfully processed event by the user from a particular
+   * partition of a consumer group in an Event Hub instance.
+   *
+   * @param sequenceNumber The sequence number of the event that you want to update the checkpoint with.
+   * @param offset The offset of the event that you want to update the checkpoint with.
+   * @return  Promise<void>.
+   */
+  updateCheckpoint(sequenceNumber: number, offset: number): Promise<void>;
+  /**
+   * @internal
+   * @ignore
+   */
+  updateCheckpoint(
```

Comment:
Do you need to specify this in the interface?

Reply:
I think I was just trying to eliminate an "ae-" error in the API extractor. I'll revisit.